src/tests/multihost/sssd/testlib/common/qe_class.py

```diff
@@ -148,6 +148,7 @@ def service_sssd(self, action):
             time.sleep(10)
             return cmd.returncode
         self.run_command('journalctl -xeu sssd.service', raiseonerr=False)
+        self.run_command('cat /var/log/sssd/sssd.log', raiseonerr=False)
```

Comment:
Do we really need `cat` for log as we're getting in artifacts?

Reply:
We are not getting it in artifacts when it matters (idmci collects it after all tests  are finished). This is to help with debugging sssd not starting. The journal seems to show only that sssd is successfully deactivated.

Reply:
okay